### PR TITLE
[stable/prometheus] Add optional extra node-exporter pod labels

### PR DESCRIPTION
**Pull request by rpiotrow**
**What this PR does / why we need it**:
Makes possible to add extra labels for node-exporter pods.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
I needed to add extra labels to enable all features in dashboards in https://grafana.com/plugins/grafana-kubernetes-app

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/prometheus/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: prometheus
-version: 6.2.1
+version: 6.2.2
```
Comment (timstoop) [inline stable/prometheus/Chart.yaml]: As this adds a new feature, please bump the minor version.

**Review by timstoop — CHANGES_REQUESTED** [review]

File: stable/prometheus/README.md [hunk 2]
```diff
@@ -167,6 +167,7 @@ Parameter | Description | Default
 `nodeExporter.extraConfigmapMounts` | Additional node-exporter configMap mounts | `[]`
 `nodeExporter.nodeSelector` | node labels for node-exporter pod assignment | `{}`
 `nodeExporter.podAnnotations` | annotations to be added to node-exporter pods | `{}`
+`nodeExporter.pod.labels` | labels to be added to node-exporter pods | `{}`
```
Comment (unguiculus) [inline stable/prometheus/README.md:174]: To be consistent with `podAnnotations`, call it `podLabels`.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/prometheus/README.md (hunk 2 above)
Comment (rpiotrow) [reply stable/prometheus/README.md:174]: What about `kubeStateMetrics.pod.labels` in https://github.com/kubernetes/charts/blob/master/stable/prometheus/templates/kube-state-metrics-deployment.yaml#L24#L26 ? Should we consider to change it as well?

Comment (gianrubio) [reply stable/prometheus/README.md:174]: @unguiculus do we have a document explaining what's the right nomenclature for labels and annotations? I don't think it's a issue to use pod.labels if they were used before by another object.

Comment (unguiculus) [reply stable/prometheus/README.md:174]: I didn't see it was used as such before. So I guess we can go with that. It's not ideal. Maybe we can fix this inconsistency in another PR. There's no specific best practice for this, but the additional level of nesting just for one value is not ideal.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**